diff --git a/frontend/src/router/index.ts b/frontend/src/router/index.ts
index 78fae81..462f7e3 100644
--- a/frontend/src/router/index.ts
+++ b/frontend/src/router/index.ts
@@ -23,10 +23,14 @@ const router = createRouter({
       name: 'history',
       component: () => import('../views/History.vue'),
     },
+    {
+      path: '/files/:fileTab?',
+      name: 'files',
+      component: () => import('../views/FilesView.vue'),
+    },
     {
       path: '/hardlinks',
-      name: 'hardlinks',
-      component: () => import('../views/HardlinkView.vue'),
+      redirect: '/files/hardlinks',
     },
     {
       path: '/search/resources',
