diff --git a/frontend/src/api/types.ts b/frontend/src/api/types.ts
index e346cb1..3bde963 100644
--- a/frontend/src/api/types.ts
+++ b/frontend/src/api/types.ts
@@ -102,6 +102,21 @@ export interface JobSettings {
   execution_provider?: string
   device_ids?: string
   processors?: string
+  face_swapper_model?: string
+  face_swapper_pixel_boost?: string
+  face_swapper_weight?: number
+  face_enhancer_model?: string
+  face_enhancer_blend?: number
+  face_enhancer_weight?: number
+  frame_enhancer_model?: string
+  frame_enhancer_blend?: number
+  face_detector_model?: string
+  face_detector_score?: number
+  face_detector_angles?: string
+  face_mask_types?: string
+  face_mask_regions?: string
+  face_mask_blur?: number
+  face_occluder_model?: string
   face_selector_mode?: string
   reference_frame_number?: number
   reference_face_position?: number
